README.md

```diff
@@ -346,9 +346,9 @@ When you call /John/12345 you will be returned:
 ```
 
 ### TemplateSwitch your JSON
-The `templateSwitch` setting uses the same structure as the `switch` setting and similar but more flexible than the `enableTemplate` feature in order to map parameter names and values from the request  into the mock response. GET and POST requests are supported including  powerful JSONPath parameter substitution even substitution into a JSON POST BODY. All you need to do is add the templateSwitch section, specify a content type for the template file, and have a matching @ in the template file.
+The `templateSwitch` setting uses the same structure as the `switch` setting and similar but more flexible than the `enableTemplate` feature in order to map parameter names and values from the request  into the mock response. GET and POST requests are supported including  powerful JSONPath parameter substitution even substitution into a JSON POST BODY. All you need to do is add the templateSwitch section, specify a content type for the template file, and have a matching @ (or in the case of unquoted numbers, use @@) in the template file.
```

Comment:
Extra space here --> 'including  powerful JSONPath' (between 'including' and 'powerful')